app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -420,10 +420,19 @@ def save_tags_generated_from_labels(entity, hashes, _target = nil)
                      :name          => label_hash[:name],
                      :value         => label_hash[:value])
     end
-    current_tags = labels.collect_concat { |label| ContainerLabelTagMapping.tags_for_label(label) }
-    mappable_tags = ContainerLabelTagMapping.mappable_tags
 
-    entity.tags = entity.tags - mappable_tags + current_tags
+    begin
+      current_tags = labels.collect_concat { |label| ContainerLabelTagMapping.tags_for_label(label) }
```

Comment:
next line also calls ContainerLabelTagMapping (the `mappable_tags` method hasn't crashed so far, but...), 
and if either fails we should skip the `entity.tags = ...` assignment.